README.md

```diff
@@ -50,6 +50,8 @@ spec:
   noCache: false
   repository: ghcr.io/k8sgpt-ai/k8sgpt
   version: v0.3.8
+  imagePullSecrets:
```

Comment:
Not everyone will use pull secrets, even if they do can you put a more generic secret name e.g. `mysecret` ? we only need an example of this in the full example of the `k8sgpt` CR

Reply:
you are right. i add k8sgpt configuration option